### DISCUSSION TITLE
Player not resuming after TTS

### DISCUSSION THREAD

**Discussion by nehasjains** [music-assistant/support#1463] [Q&A] [answered]
Hello Friends,

Playback is not resuming after a TTS, it has to be manually played again.

For every case PLAYBACK is not resumed after TTS
1. Playing songs from local file system.
2. Playing from YouTube.
3. Playing from radio.

I have tried this with google_translate, Pico tts.

For tts I am targeting the media player created by MA which is my Google nest mini.

Am I missing something? Is there any setting that is to be enabled ?

Using Homeassistant on raspberrypi 4, installation type is supervised. MA installed is latest beta.

Kindly guide.

**Comment by OzGav** [comment, marked as answer]
This functionality is yet to be reimplemented. Your patience is requested.

**Reply by nehasjains** [reply]
Thank you for your kind reply.

**Comment by OzGav** [comment]
This is implemented
